modules/ROOT/partials/configuration/exportword.adoc

```diff
@@ -8,6 +8,9 @@ The {pluginname} feature is currently only available exclusively for `on-premise
 
 NOTE: The {pluginname} feature is currently only available `on-premise` and requires the `exportword_service_url` option to be configured. link:https://www.tiny.cloud/contact/[Contact us] if you require this feature.
 
+[TIP]
+Update the `exportword_service_url` configuration in your {productname} setup to point to your host. For instance, if your host is `mycustomhost.com`, the URL will be `https://mycustomhost.com/v2/convert/html-docx`.
```

Comment:
I guess technically this could be just a `https://mycustomhost.com/` but now it really doesn't matter. Since it will now always just add `v2/convert/html-docx` to the end of all URLs.

Reply:
My thoughts was that we would try to provide the most accurate complete URL, but i guess it doesn't really matter now that we update the URL before the POST request is sent.